src/components/DpResettableInput/DpResettableInput.vue

```diff
@@ -128,13 +128,12 @@ export default {
     }
   },
 
-  watch: {
-    value: function () {
-      this.currentValue = this.value
-    }
-  },
-
```

Comment:
We still need this watcher, because resetting is handled in the parent via the value.